### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by xpwn3rx** [ScoopInstaller/Main#4253] [Q&A]
#### Current Behavior
Running scoop update * or scoop update fzf in PowerShell 7.3 release does not work

#### Expected Behavior
I expected it to successfully update

#### Additional context/output
╰─❯ scoop update fzf
fzf: 0.34.0 -> 0.35.1
Updating one outdated app:
Updating 'fzf' (0.34.0 -> 0.35.1)
Downloading new version
fzf-0.35.1-windows_amd64.zip (1.3 MB) [=======================================================================] 100%
Checking hash of fzf-0.35.1-windows_amd64.zip ... ok.
Uninstalling 'fzf' (0.34.0)
Removing shim 'fzf.shim'.
Removing shim 'fzf.exe'.
Unlinking ~\scoop\apps\fzf\current
Installing 'fzf' (0.35.1) [64bit] from main bucket
Loading fzf-0.35.1-windows_amd64.zip from cache
Extracting fzf-0.35.1-windows_amd64.zip ... done.
Linking ~\scoop\apps\fzf\current => ~\scoop\apps\fzf\0.35.1
Creating shim for 'fzf'.
Can't shim 'fzf.exe': File doesn't exist.
╭─ pwsh  socce@PWn3R-Home  C:\                                                                         10:18:45 
╰─❯ scoop update
Updating Scoop...
Updating 'main' bucket...
Scoop was updated successfully!
╭─ pwsh  socce@PWn3R-Home  C:\                                                                         10:19:16 
╰─❯ scoop update *
curl: 7.85.0_9 -> 7.86.0_2
fzf: 0.34.0 -> 0.35.1
neovim: 0.8.0 -> 0.8.1
nvm: 1.1.9 -> 1.1.10
Updating 4 outdated apps:
Updating 'curl' (7.85.0_9 -> 7.86.0_2)
Downloading new version
curl-7.86.0_2-win64-mingw.tar.xz (6.0 MB) [===================================================================] 100%
Checking hash of curl-7.86.0_2-win64-mingw.tar.xz ... ok.
Uninstalling 'curl' (7.85.0_9)
Removing shim 'curl.shim'.
Removing shim 'curl.exe'.
Unlinking ~\scoop\apps\curl\current
Installing 'curl' (7.86.0_2) [64bit] from main bucket
Loading curl-7.86.0_2-win64-mingw.tar.xz from cache
Extracting curl-7.86.0_2-win64-mingw.tar.xz ... Invoke-ExternalCommand: C:\Users\socce\scoop\apps\scoop\current\lib\decompress.ps1:45
Line |
  45 |      $Status = Invoke-ExternalCommand $7zPath $ArgList -LogPath $LogPa …
     |                                       ~~~~~~~
     | Cannot validate argument on parameter 'FilePath'. The argument is null or empty. Provide an argument that is not
     | null or empty, and then try the command again.
Failed to extract files from C:\Users\socce\scoop\apps\curl\7.86.0_2\curl-7.86.0_2-win64-mingw.tar.xz.
Log file:
  ~\scoop\apps\curl\7.86.0_2\7zip.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=curl%407.86.0_2%3a+decompress+error

### System details

**Windows version:** [e.g. 7, 8, 10, 11]
Windows 11

**OS architecture:** [e.g. 32bit, 64bit, arm64]
64bit

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
Major  Minor  Patch  PreReleaseLabel BuildLabel
-----  -----  -----  --------------- ----------
7      3      0

**Additional software:** [(optional) e.g. ConEmu, Git]
Git 
1
#### Scoop Configuration

```json
//# Your configuration here
╰─❯ cat C:\Users\socce\.config\scoop\config.json
{
  "last_update": "2022-12-18T10:19:16.4365897-07:00",
  "scoop_branch": "master",
  "scoop_repo": "https://github.com/ScoopInstaller/Scoop"
}

**Comment by HUMORCE** [comment]
Cannot reproduce.

Please retry with Windows PowerShell 5.1 / PowerShell Core 7.3.1(Latest stable) or 7.2.8(LTS)
